less/home-page.less

```diff
@@ -58,12 +96,28 @@
       }
 
       .btn {
+        .btn-mxn(10px 20px, 110px);
+
+        @media @mobile {
+          .mobile-width(100%);
+          padding: 20px 0;
+        }
+      }
+
+      .btn:hover {
+        .btn-hvr;
       }
     }
 
     .walk-img {
       width: 46%;
       margin-left: 4%;
+
+      @media @mobile {
+        .mobile-width(90%);
+        order: -1;
```

Comment:
Clever use of just modifying the order here to move the image to where you wanted it (above the text content). 

This could have been done using `flex-direction` of column reverse, which would have automatically changed the orientation of the children elements to vertical, and reverse their order.

 I like the method you used since this will be the exception, and there is no reason to rewrite the rule. 

Multiple ways to skin a cat, I just wanted to expose you to another here.